vizro-core/src/vizro/static/css/layout.css

```diff
@@ -117,7 +117,7 @@
 
 .icon-button {
   background-color: var(--surfaces-bg-02);
-  width: 64px;
+  width: 63px;
```

Comment:
This feels kind of hacky - is there any other way of doing it? Maybe there's not, in which case this is fine, but do we have any other way to solve this that we do elsewhere in the codebase?

Reply:
I forgot to change it when testing. I've fixed it now by defining width: 100% instead of hardcoding px. Thanks for pointing it out.